### PR TITLE
docs: provide download links to Artifactory CE

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: uploading_packages/artifactory/artifactory_ce.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Artifactory Community Edition for C/C++
 =======================================
 
-Artifactory Community Edition (CE) for C/C++ is the recommended server for development and hosting private
+`Artifactory Community Edition (CE) for C/C++ <https://jfrog.com/community/open-source/>`_ is the recommended server for development and hosting private
```
Comment (memsharded) [inline uploading_packages/artifactory/artifactory_ce.rst:6]: This page is a bit confusing, and shows different solutions, not that evident to the newbie what thing needs to be download, plus it also features some things like ``IDE Integrations``, that might lead to confusion, thinking there are Conan integrations with those IDEs? 

Plus that page, what it does is to redirect back to https://conan.io/downloads.html, so maybe point to that one directly? One jump less

Comment (prince-chrismc) [reply uploading_packages/artifactory/artifactory_ce.rst:6]: I put that one a touch lower when it says "you can download it by" but I was trying to add a generic marketing page about the Community Edition but there's not many picks

Maybe just not put anything?

Comment (prince-chrismc) [reply uploading_packages/artifactory/artifactory_ce.rst:6]:
```suggestion
Artifactory Community Edition (CE) for C/C++ is the recommended server for development and hosting private
```

Comment (memsharded) [reply uploading_packages/artifactory/artifactory_ce.rst]: Yeah, maybe dropping it is the best.

**Review by RubenRBS — APPROVED** [review]

File: uploading_packages/artifactory/artifactory_ce.rst (hunk 1 above)
Comment (prince-chrismc) [reply uploading_packages/artifactory/artifactory_ce.rst]: 👍

File: uploading_packages/artifactory/artifactory_ce.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Running Artifactory CE
 ----------------------
 
 There are several ways to download and run Artifactory CE. The simplest one might be to download and unzip the
-designated zip file, though other installers, including also installing from a Docker image.
+designated zip file, though other installers, including also installing from a Docker image. The _`Download Page <https://conan.io/downloads.html>`_ has a link for you to follow.
```
Comment (memsharded) [inline uploading_packages/artifactory/artifactory_ce.rst:18]: This is breaking CI, wrong link syntax?

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**